packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -634,6 +649,7 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
                 <ComponentWrapper
                     key={index}
                     index={index}
+                    background={this.state.theme.background || null}
```

Comment:
If theme is undefined this will error because it will try to look up background on an undefined property. You can do `this.state.theme && this.state.theme.background ? this.state.theme.background : null`

I would also make the same change on line 620.